packages/core/src/js/feedback/FeedbackForm.tsx

```diff
@@ -80,7 +80,16 @@ export class FeedbackForm extends React.Component<FeedbackFormProps, FeedbackFor
 
     return (
     <View style={styles.container}>
-      <Text style={styles.title}>{text.formTitle}</Text>
+      <View style={styles.titleContainer}>
+          <Text style={styles.title}>{text.formTitle}</Text>
+          {config.showBranding && (
+            <Image
+              source={{ uri: 'https://sentry-brand.storage.googleapis.com/sentry-glyph-black.png' }}
```

Comment:
Let's include logo locally.

I'm not sure who in Sentry owns this storage but it could be gone any time.

Reply:
That's a good point @krystofwoldrich 👍 
I actually tried this (https://github.com/getsentry/sentry-react-native/pull/4357/commits/61e4d4daca6b6233c8936fc097601c5e648d5302) but run into errors with packaging. I'll revisit my approach.

Reply:
Updates with 727a423a09dec83c42937fc4d6da2dd43a83a3d6 to include the logo as a base64 encoded image locally.
I've [tried](https://dev.to/vpicone/how-to-ship-static-assets-with-your-react-native-library-22pf) including it as a resource in the [assets folder](https://github.com/getsentry/sentry-react-native/tree/main/packages/core/assets) but run into issues with rendering on Android and the end packaging in the SDK.